constants/license_classes.json

```diff
@@ -174,6 +174,11 @@
         "0",
         "0"
     ],
+    "NVIDIA License": [
```

Comment:
I think we should add this to custom licenses, as it is a NVIDIA license, but not the only type of NVIDIA license, if I understand correctly?

Reply:
Probably, I couldn't find a link for the definition.